src/libsyntax_ext/deriving/custom.rs

```diff
@@ -28,6 +28,9 @@ impl<'a> Visitor<'a> for MarkAttrs<'a> {
             mark_known(attr);
         }
     }
+
+    fn visit_mac(&mut self, _mac: &Mac) {
+    }
```

Comment:
nit: I believe using a single line for empty functions (i.e. `fn f(...) {}`) is more idiomatic, at least in the compiler.